membershipextras.civix.php

```diff
@@ -269,10 +269,10 @@ function _membershipextras_civix_civicrm_managed(&$entities) {
       if (empty($e['module'])) {
         $e['module'] = E::LONG_NAME;
       }
+      $entities[] = $e;
```

Comment:
I think this is a bug that's fixed on latest versions of civix. I believe this line should be removed from here and added to line 275, as it was before.